js/apps/account-ui/src/applications/Applications.tsx

```diff
@@ -227,14 +228,7 @@ const Applications = () => {
                         {t("accessGrantedOn")}
                       </DescriptionListTerm>
                       <DescriptionListDescription>
-                        {new Intl.DateTimeFormat("en", {
-                          year: "numeric",
-                          month: "long",
-                          day: "numeric",
-                          hour: "numeric",
-                          minute: "numeric",
-                          second: "numeric",
-                        }).format(application.consent.createdDate)}
+                        {formatDate(new Date(application.consent.createdDate))}
```

Comment:
@edewit I am not quite sure how to get this piece of UI to show up. Could you test this, or tell me how to test it? I need to know if we need to multiply this value by a 1000, like in the other places where this bug was popping up.

Reply:
you need to enable consent on the client:
![image](https://github.com/keycloak/keycloak/assets/51133/d33c5e64-c0f5-4060-a5eb-887e7abe5309)
Then you get it, but it seems that we are missing some translations:
![image](https://github.com/keycloak/keycloak/assets/51133/d8bfbc6e-faea-4d68-91f6-ba9a5c81f1f0)

Reply:
We should certainly fix that one, can you log an issue for it?